Connect-with-Exasol/openresty-odbc-connection.md

```diff
@@ -0,0 +1,88 @@
+# OpenResty Exasol ODBC connection
+
+## Background
+
+Connecting to Exasol from OpenResty lets you build fast APIs or dynamic web endpoints directly inside Nginx using Lua.
+This enables you to query Exasol on the fly and return results to clients without an extra backend layer.
+
+## Steps
+
+Below are steps to connect to Exasol from Openresty.
+
+* Install openresty [Openresty Installation manual](https://openresty.org/en/linux-packages.html#ubuntu)
+* install Luarocks
+
+```shell
+wget https://github.com/luarocks/luarocks/archive/v3.0.0.tar.gz
+
+tar zxvf v3.0.0.tar.gz
+
+./configure --prefix=/usr/local/openresty/luajit/ --with-lua=/usr/local/openresty/luajit/ --lua-suffix=jit --with-lua-include=/usr/local/openresty/luajit/include/luajit-2.1
+
+sudo make build
+
+sudo make install
+```
+
+* install ODBC package in luarocks
+
+```shell
+luarocks install odbc
+```
+
+* install Exasol ODBC driver, create DSN for Exasol DB [ODBC Drivers for Linux/FreeBSD | Exasol Documentation](https://docs.exasol.com/db/latest/connect_exasol/drivers/odbc/odbc_linux.htm)
+
+* Execute connection test scripts
+
+ odbctest.sh
+
+```shell
+# set -x
+# ORHOME is the path to the OpenResty install
+export ORHOME="/usr/local/openresty"
+export ODBCINI="${ORHOME}/odbc/.odbc.ini"
+export LD_LIBRARY_PATH="${ORHOME}/luajit/lib"
+export PATH="$PATH:${ORHOME}/bin/"
+resty odbctest.lua $1
+echo "Retun code from resty $?"
+```
```

Comment:
````suggestion
    ```shell
    # set -x
    # ORHOME is the path to the OpenResty install
    export ORHOME="/usr/local/openresty"
    export ODBCINI="${ORHOME}/odbc/.odbc.ini"
    export LD_LIBRARY_PATH="${ORHOME}/luajit/lib"
    export PATH="$PATH:${ORHOME}/bin/"
    resty odbctest.lua $1
    echo "Retun code from resty $?"
    ```
````